packages/components/src/components/NewModal/NewModalBackdrop.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import FocusLock from 'react-focus-lock';
+import { zIndices, spacings } from '@trezor/theme';
+import { createPortal } from 'react-dom';
+
+import { useModalTarget } from './NewModalProvider';
+import type { UIHorizontalAlignment, UIVerticalAlignment } from '../../config/types';
+
+export type ModalAlignment = { x: UIHorizontalAlignment; y: UIVerticalAlignment };
+
+export type NewModalBackdropProps = {
+    onClick?: () => void;
+    children?: ReactNode;
+    alignment?: ModalAlignment;
+    padding?: number;
+};
+
+const Wrapper = styled.div<{ $alignment: ModalAlignment; $padding: number }>`
+    position: absolute;
+    z-index: ${zIndices.modal};
+    inset: 0;
+    display: flex;
+    flex-direction: column;
+    padding: ${({ $padding }) => $padding}px;
+    overflow: auto;
+
+    /* backdrop-filter does not work in Firefox, use darker background instead */
```

Comment:
<img width="1371" alt="image" src="https://github.com/user-attachments/assets/8bebd64a-d2b1-4c83-b862-775eb6a8492c">

Are you sure?

btw could you add also `-webkit-backdrop-filter` for Safari?

Reply:
Will do. Also taken verbatim from the old modal